amid/luna25.py

```diff
@@ -0,0 +1,138 @@
+from datetime import datetime
+from functools import cached_property
+from typing import Dict, NamedTuple, Sequence
+
+import numpy as np
+import pandas as pd
+import SimpleITK as sitk
+
+from .internals import Dataset, field, licenses, register
+
+
+class NoduleBlock(NamedTuple):
+    image: np.ndarray
+    metadata: Dict
+
+
+class LUNA25Nodule(NamedTuple):
+    coords: Sequence[float]
+    lesion_id: int
+    annotation_id: str
+    nodule_id: str
+    malignancy: bool
+    center_voxel: Sequence[float]
+    bbox: np.ndarray
+
+
+@register(
+    body_region='Chest',
+    license=licenses.CC_BY_40,
+    link='https://luna25.grand-challenge.org/',
+    modality='CT',
+    prep_data_size=None,
+    raw_data_size=None,
+    task='Lung nodule malignancy risk estimation',
+)
+class LUNA25(Dataset):
+    """
+    The LUNA25 Challenge dataset is a comprehensive collection designed to support
+    the development and validation of AI algorithms for lung nodule malignancy risk
+    estimation using low-dose chest CT scans. In total, it contains 2120 patients
+    and 4069 low-dose chest CT scans, with 555 annotated malignant nodules and
+    5608 benign nodules (3762 unique nodules, 348 of them are malignant).
+    The dataset was acquired in participants who enrolled in the
+    National Lung Cancer Screening Trial (NLST) between 2002 and 2004 in
+    one of the 33 centers in the United States.
+
+    Parameters
+    ----------
+    root : str, Path, optional
+        path to the folder containing the raw downloaded archives.
```

Comment:
The code operates with unpacked files, however the data is shipped in archives. I think that the instructions on data preparation should be provided, e.g. on L57 the code uses `luna25_images` constant. Also, would be cool to add a check inside a constructor to ensure that all the files are downloaded and extracted properly (like [here](https://github.com/neuro-ml/amid/blob/master/amid/lidc/dataset.py#L75)).